src/main/java/mineverse/Aust1n46/chat/versions/VersionHandler.java

```diff
@@ -67,10 +67,22 @@ public static boolean is1_19() {
 	}
 
 	public static boolean isUnder_1_19() {
-		return !SERVER_VERSION.isAtLeast(MinecraftVersion.WILD_UPDATE);
+		if (SERVER_VERSION.getMajor() < 1) {
+			return true;
+		}
+		return SERVER_VERSION.getMajor() == 1 && SERVER_VERSION.getMinor() < 19;
 	}
 
 	public static boolean isAbove_1_19() {
-		return !is1_19() && SERVER_VERSION.isAtLeast(MinecraftVersion.WILD_UPDATE);
+		if (SERVER_VERSION.getMajor() < 1) {
+			return false;
+		}
+		if (SERVER_VERSION.getMajor() > 1) {
+			return true;
+		}
+		if (SERVER_VERSION.getMinor() > 19) {
+			return true;
+		}
+		return SERVER_VERSION.getMinor() == 19 && SERVER_VERSION.getBuild() > 0;
```

Comment:
`private static final MinecraftVersion MC1_19_1 = new MinecraftVersion(1, 19, 1);`

```suggestion
		return SERVER_VERSION.isAtLeast(MC1_19_1);
```